.github/workflows/python_dbapi_wheels.yml

```diff
@@ -0,0 +1,104 @@
+name: "Build Python DB API Driver Wheels"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build_source_dist:
+    name: Build source distribution
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.13'
+
+      - name: Install setuptools
+        run: python -m pip install setuptools
+
+      - name: Copy C++ sources
+        run: cp -r modules/platforms/cpp modules/platforms/python/
+
+      - name: Build source distribution
+        working-directory: modules/platforms/python
+        run: python setup.py sdist --formats=gztar --dist-dir ./dist
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: source-distributions
+          path: modules/platforms/python/dist/*
+
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # macos-13 is an intel runner, macos-14 is apple silicon
+        os: [ubuntu-latest, ubuntu-24.04-arm, windows-latest, macos-13, macos-14]
```

Comment:
I think we should not use `latest`, specify a version for all entries.

Reply:
Good point. Maybe I should not even use latest fixed version and use more widely adopted instead.